app/src/scripts/common/partials/comment-tree.jsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react'
+import Reflux from 'reflux'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+import { withRouter } from 'react-router-dom'
+import Comment from './comment.jsx'
+import WarnButton from '../forms/warn-button.jsx'
+
+class CommentTree extends Reflux.Component {
```

Comment:
Is this supposed to be a Reflux component?

Reply:
i) deleted comments now show the [deleted] tag above the comment. admins still have the ability to remove / edit / reply to deleted comments

ii) code syntax select dropdown is now tied to the syntax state of the comment. while this is only single syntax per comment, it now works more along the lines of intuition.

iii) the styling has been tightened up